server/sonar-web/src/main/js/apps/project-admin/store/actions.js

```diff
@@ -102,3 +104,30 @@ export const deleteProjectLink = (projectKey, linkId) => ({
   projectKey,
   linkId
 });
+
+export const RECEIVE_PROJECT_MODULES = 'RECEIVE_PROJECT_MODULES';
+const receiveProjectModules = (projectKey, modules) => ({
+  type: RECEIVE_PROJECT_MODULES,
+  projectKey,
+  modules
+});
+
+// TODO pagination
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1135)